tests/e2e/privacy.spec.js

```diff
@@ -25,6 +28,10 @@ async function updatePrivacySettings(name, value) {
   }
 
   await expect(toggle).toHaveElementProperty('value', value);
+
+  // Allow background process to update the settings
+  // E.g. reload engines / DNR rules
+  await browser.pause(2000);
```

Comment:
would be best to create a github issue to provide visual feedback to the settings page for which those tests could wait to confirm the action was successful (or not successful)

Reply:
I will be hard, as those are listeners to the options, which also triggers in other situations. For example merging engines happens for many reasons, but after switching some of the toggles, it happens.

Reply:
gotcha, lets discuss if we don't want to change this architecture as having some sorts of visual feedback may be useful for some users, not only for testing. I would definitely want to get confirmation if DNR ruleset got applied successfully for example.